randovania/games/prime2/logic_database/Torvus Bog.txt

```diff
@@ -1579,10 +1583,22 @@ Extra - in_dark_aether: False
   * Extra - dock_name: South
   > Door to Grove Access
       All of the following:
-          Morph Ball and Space Jump Boots
+          Morph Ball
           Any of the following:
-              Boost Ball
-              Screw Attack and After Torvus Grove Tree and Standable Terrain (Advanced)
+              All of the following:
+                  Boost Ball
+                  Any of the following:
+                      Space Jump Boots
+                      All of the following:
+                          Morph Ball Bomb
+                          Any of the following:
+                              All of the following:
+                                  # https://www.youtube.com/watch?v=JTfbOmmyfZ8
+                                  Bomb Space Jump (Expert)
+                                  Combat/Scan Dash (Intermediate) or Slope Jump (Advanced)
```

Comment:
Do you have  a video for the slope jump method? Or is this supposed to be an and?